packages/fxa-settings/src/components/GetDataTrio/index.tsx

```diff
@@ -13,13 +13,13 @@ import { FtlMsg } from 'fxa-react/lib/utils';
 
 export type DownloadContentType =
   | 'Firefox account recovery key'
```

Comment:
If the other content types/strings are no longer used, we could add a note to remove in the follow up ticket or in FXA-7419 (recovery key cleanup)

Reply:
In have filed a sun setting ticket specifically for facing out old strings. https://mozilla-hub.atlassian.net/browse/FXA-8474